src/main/java/io/github/thebusybiscuit/slimefun4/api/gps/TeleportationManager.java

```diff
@@ -161,11 +161,15 @@ public int getTeleportationTime(int complexity, @Nonnull Location source, @Nonnu
             return 100;
         }
 
-        int speed = 50_000 + complexity * complexity;
-        int unsafeTime = Math.min(4 * distanceSquared(source, destination) / speed, 40);
+        long speed = 50_000L + (long)complexity * complexity;
+        int distance = 4 * distanceSquared(source, destination), time = 1;
```

Comment:
```suggestion
        int distance = 4 * distanceSquared(source, destination)
        int time = 1;
```
I don't think we ever declare two variables on the same row so for consistency this makes more sense

Reply:
Actually let me just commit the change